src/css/controls/imports/settings-menu.less

```diff
@@ -135,6 +136,10 @@
     .jw-settings-close {
         margin-left: auto;
     }
+
+    div {
```

Comment:
lots of generic selectors, might bite us in future but seems safe enough given they are localized in these blocks